lib/paymentrequest.py

```diff
@@ -53,6 +53,21 @@
 from .util import FileImportFailed, FileImportFailedEncrypted
 from .transaction import Transaction
 
+def _(message): return message
+
+# status of payment requests
+PR_UNPAID  = 0
+PR_EXPIRED = 1
+PR_UNKNOWN = 2     # sent but not propagated
+PR_PAID    = 3     # send and propagated
+
+pr_tooltips = {
+    PR_UNPAID:_('Pending'),
+    PR_UNKNOWN:_('Unknown'),
+    PR_PAID:_('Paid'),
+    PR_EXPIRED:_('Expired')
+}
+
```

Comment:
You should do `del _` here for clairty.  The `_` function is used as a decorator so that the text can get picked up by gnu `gettext`.   If you `del` it right away it makes anyone reading that section realize this is the only place in the file this happens and perhaps why it happens.

Reply:
Done